app/src/main/java/com/example/parkingsystemjava/utils/ReservationErrorCodes.java

```diff
@@ -0,0 +1,5 @@
+package com.example.parkingsystemjava.utils;
+
+public enum ReservationErrorCodes {
+    MISSING_ENTRY, MISSING_EXIT, WRONG_PARKING_LOT, MISSING_KEYCODE, RESERVATION_OVERLAP, OK
+}
```

Comment:
pls format the enum class to:
public enum ReservationErrorCodes {
    MISSING_ENTRY,
    MISSING_EXIT, 
    WRONG_PARKING_LOT, 
    MISSING_KEYCODE,
    RESERVATION_OVERLAP, OK
}

Reply:
done